frontend/src/api/authRequest.test.tsx

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable max-nested-callbacks */
+import {AuthApiClient} from './authRequest'
+import { HttpClient } from './HttpClient'
+
+jest.mock('./HttpClient')
+
+describe('use of AuthApiClient', () => {
+  beforeEach(() => {
+    HttpClient.mockClear()
+  })
+  
+  it('We can check if authClient called the class constructor', () => {
+    const authClient = new AuthApiClient()
+    expect(HttpClient).toHaveBeenCalledTimes(1)
+  })
+
+  it('AuthClent should be able to call new() on HttpClient', () => {
```

Comment:
This test does not check what it says (this is done by previous test).
Mock methods of `HttpClient` (not the whole class), and test if the methods of `authClient` are using methods of `HttpClient`